source/_integrations/dyson_local.markdown

```diff
@@ -0,0 +1,30 @@
+---
+title: Dyson Local
+description: Instructions on how to integrate Dyson Local into Home Assistant.
+ha_category:
+  - Binary Sensor
+  - Sensor
+  - Vacuum
+ha_iot_class: Local Push
+ha_release: '2021.9'
+ha_config_flow: true
+ha_domain: dyson_local
```

Comment:
I am confused why this adds a new domain? Why not integrate this with the existing Dyson integration?